src/theme/base/opacity.ts

```diff
@@ -0,0 +1,20 @@
+const opacity = {
+  0: 0,
```

Comment:
I think the key should be a string rather than a number

Reply:
done

Reply:
I don't think terms like tiny, miniature, or petite are appropriate in relation to opacity.

What do you think about:

Light
Bright
Radiant
Shimmering
Gleaming
Dazzling
Shining
Glimmering
Illuminated
Twilight
Gloomy
Somber
Dark

Reply:
done